src/macaron/slsa_analyzer/specs/package_registry_data.py

```diff
@@ -0,0 +1,46 @@
+# Copyright (c) 2023 - 2023, Oracle and/or its affiliates. All rights reserved.
+# Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl/.
+
+
+"""This module contains data related to one package registry that is matched against a repository."""
+
+from dataclasses import dataclass, field
+
+from macaron.slsa_analyzer.asset import Asset
+from macaron.slsa_analyzer.build_tool import BaseBuildTool
+from macaron.slsa_analyzer.package_registry import PackageRegistry
+from macaron.util import JsonType
+
+
+@dataclass
+class PackageRegistryData:
+    """This class contains data for one package registry that is matched against a repository.
+
+    Attributes
+    ----------
+    build_tool : BaseBuildTool
+        The build tool matched against the repository.
+
+    package_registry : PackageRegistry
+        The package registry matched against the repository. This is dependent on the build tool detected.
+
+    latest_version : str | None
+        The latest version of the artifact found on the registry.
+
+    provenance_assets : list[dict]
```

Comment:
Should this be `list[Asset]` ?

Reply:
This is now obsolete due to 93beee3fdd0cbbb0f3fc90b77339dcfea22950f7.